### DISCUSSION TITLE
2023 WHO Mutation Catalogue

### DISCUSSION THREAD

**Discussion by taranewman** [jodyphelan/TBProfiler#314] [Q&A] [answered]
Hi Jody,

I was just wondering if you will be looking into the 2023 WHO catalogue of mutations and updating the tbdb with any new data from there?

(https://github.com/GTB-tbsequencing/mutation-catalogue-2023) 

Thank you!
Tara

**Comment by jodyphelan** [comment, marked as answer]
Hi @taranewman 

Yes this is currently in progress. Hope to have it released before the end of the year!
